cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -434,25 +434,18 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) caerrors.AutoscalerErr
 	// Check if there has been a constant difference between the number of nodes in k8s and
 	// the number of nodes on the cloud provider side.
 	// TODO: andrewskim - add protection for ready AWS nodes.
-
-	// FORK-CHANGE: Commented this code as it removes `Registered but long not Ready` nodes which causes issues like scaling below minimum size and removing ready nodes during meltdown scenario
```

Comment:
Never undo the `FORK-CHANGE`. Can you please revert this? These are the changes done in the core CA by us. We need to keep them. Please revert the changes at other similar places